UI-extra/price.dust

```diff
@@ -0,0 +1,13 @@
+<div class="price">
+    {?oldPrice}
+        <s class="grey-color">{oldprice}</s>
+        <span>{currentprice}</span>
+        {?savePrice}
```

Comment:
I think you only want to show one set of prices if I'm correct.  So in this case, I would probably do something like

`
{?oldPrice}
<s class="grey-color">{oldPrice}</s>
{?saveprice}<span class="green-color">{saveprice}</span>{/saveprice}
<span {?saveprice}class="red-color"{/saveprice}>{currentPrice}</span>
{:else}
 ...
{/oldPrice}
`